app/assets/javascripts/spotlight/multi_image_selector.js

```diff
@@ -103,7 +103,7 @@
     }
     function addThumbsToList(){
       $.each(image_versions, function(i){
-        var listItem = $('<li><a href="javascript:;"><img data-iiif-tilesource="' + image_versions[i]['tilesource'] + '" data-full-image="' + image_versions[i]['full'] +'" data-src="' + image_versions[i]['thumb'] +'" /></a></li>');
```

Comment:
It turns out that we didn't actually need this attribute.  We pass the tilesource directly when we parse the manifest.